layers/meta-balena-jetson/recipes-kernel/linux/jetson-dtbs.bb

```diff
@@ -36,6 +36,17 @@ do_install_jetson-tx2() {
 	install -m 0644 ${DEPLOY_DIR_IMAGE}/tegra186-quill-p3310-1000-c03-00-base.dtb ${D}/boot/tegra186-quill-p3310-1000-c03-00-base.dtb
 	install -m 0644 ${WORKDIR}/tegra186-tx2-cti-ASG001-revG+.dtb ${D}/boot/tegra186-tx2-cti-ASG001-revG+.dtb
 }
+do_install_jetson-tx2-4gb() {
+	install -d ${D}/boot/
+	install -m 0644 ${WORKDIR}/tegra186-tx2-6.dtb ${D}/boot/tegra186-tx2-6.dtb
+	install -m 0644 ${WORKDIR}/tegra186-tx2-cti-ASG006-IMX274-6CAM.dtb ${D}/boot/tegra186-tx2-cti-ASG006-IMX274-6CAM.dtb
+	install -m 0644 ${WORKDIR}/tegra186-tx2-cti-ASG001-USB3.dtb ${D}/boot/tegra186-tx2-cti-ASG001-USB3.dtb
+	install -m 0644 ${WORKDIR}/tegra186-tx2-aetina-n510-p3489-0888-a00-00-base.dtb ${D}/boot/tegra186-tx2-aetina-n510-p3489-0888-a00-00-base.dtb
+	install -m 0644 ${WORKDIR}/tegra186-tx2-aetina-n310-p3489-0888-a00-00-base.dtb ${D}/boot/tegra186-tx2-aetina-n310-p3489-0888-a00-00-base.dtb
+	install -m 0644 ${WORKDIR}/tegra186-tx2-blackboard.dtb ${D}/boot/tegra186-tx2-blackboard.dtb
+	install -m 0644 ${DEPLOY_DIR_IMAGE}/tegra186-quill-p3489-0888-a00-00-base.dtb ${D}/boot/tegra186-quill-p3489-0888-a00-00-base.dtb
```

Comment:
Only tegra186-quill-p3489-0888-a00-00-base.dtb is necessary when extending the existing machine, for instance:

```
do_install_append_tx2-4gb() {
    install -m 0644 ${DEPLOY_DIR_IMAGE}/tegra186-quill-p3489-0888-a00-00-base.dtb ${D}/boot/tegra186-quill-p3489-0888-a00-00-base.dtb
}
```